bin/_mkproject.bat

```diff
@@ -24,8 +24,11 @@ if "%2"=="" goto :missing_name
 if "%3"=="" goto :missing_silo
 
 set PROJECTDIR=%1%2
-set curPROJECT=%1
-set curPROJECTNAME=%2
+:: This is a variable for the function back.bat.
+set CURRENTPROJECT=%1
```

Comment:
Did we say to remove "current" from these?

Also there is one variable with an underscore, whereas the others do not have any underscores. I would either make each of them have it, or none of them.